static/.htaccess

```diff
@@ -418,469 +430,4 @@ RewriteRule ^docs/components/modeler/web-modeler/save-and-deploy/?$ /docs/compon
 RewriteRule ^docs/next/components/modeler/web-modeler/start-instance/?$ /docs/next/components/modeler/web-modeler/run-or-publish-your-process/ [R=301,L]
 RewriteRule ^docs/next/components/modeler/web-modeler/save-and-deploy/?$ /docs/next/components/modeler/web-modeler/run-or-publish-your-process/#deploy-a-process [R=301,L,NE]
 
-# Modeler start instance and deploy page renaming and merging 8.2
```

Comment:
There's a lot of removals from this point forward. All of these rules at the bottom were introduced prior to 8.3 shipping, which I think means that they were part of 8.2, and can be sunsetted with that version.

There are two rules which aren't versioned, which I moved to above sections. (One for the blog, one for slashes at the end of assets.)